python/HeaderService/hslib_salobj.py

```diff
@@ -0,0 +1,500 @@
+# This file is part of HeaderService
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import HeaderService
+import HeaderService.hutils as hutils
+import HeaderService.hscalc as hscalc
+import logging
+import os
+import sys
+import socket
+import lsst.ts.salobj as salobj
+import asyncio
+import time
+
+LOGGER = logging.getLogger(__name__)
+
+
+class HSWorker(salobj.BaseCsc):
```

Comment:
In this package I see three classes called HSWorker (though this seems to be the main one, and the others are simple examples). Please consider tweaking the names of the classes in the example files to avoid confusion.